gammapy/estimators/map/excess.py

```diff
@@ -86,13 +86,16 @@ class ExcessMapEstimator(Estimator):
         Confidence level for the asymmetric errors expressed in number of sigma.
     n_sigma_ul : float
         Confidence level for the upper limits expressed in number of sigma.
+    n_sigma_sensitivity : float
+        Confidence level for the sensitivity expressed in number of sigma.
```

Comment:
There are other options needed when computing the sensitivity, the minimal number of gamma-rays and the fraction of background counts above which the number of gamma-rays is (ie eta for which Ng>eta*Nkg).

Reply:
I added `gamma_min_sensitivity`, `bkg_syst_fraction_sensitivity and `apply_threshold_sensitivity` options.
``bkg_syst_fraction_sensitivity` and `gamma_min_sensitivity` are used in the sensitivity computation only if  `apply_threshold_sensitivity` is True. The default is set to False which match the behaviour used for the HGPS catalogue.